src/pages/akcie.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { isFuture } from 'date-fns';
+
+import { Page } from 'components/Page';
+import { FBEventsList } from 'components/FBEventsList';
+import { graphql } from 'gatsby';
+
+export default ({
+  data: {
+    allFacebookEvents: { edges: events },
+  },
+}) => (
+  <Page title="Akcie" url="/akcie">
+    <div className="font-bold text-xl mb-1">Pripravované akcie</div>
+    {events && <FBEventsList events={events} filter={(event => isFuture(new Date(event.node.start_time)))} />}
+    <div className="font-bold text-xl mb-1">Minulé akcie</div>
+    {events && <FBEventsList events={events} filter={(event => !isFuture(new Date(event.node.start_time)))} />}
+  </Page>
+);
+
+export const query = graphql`
+  {
+    allFacebookEvents {
+      edges {
+        node {
+          id
+          name
+          description
```

Comment:
Would be great if we could truncate description [like excerpts](https://www.gatsbyjs.org/docs/graphql-reference/#excerpt) instead of the `react-dotdotdot` package.

Reply:
I don't see such support in queries from `gatsby-source-facebook`. I can only see excerpt in fields from `frontmatter`.

This is the `allFacebookEvents` schema:
![Screen Shot 2019-04-06 at 19 12 54](https://user-images.githubusercontent.com/6513576/55672848-2af19d80-58a0-11e9-9d5d-a7826ea7bdfd.jpg)